SimTracker/SiStripDigitizer/plugins/SiStripDigitizerAlgorithm.cc

```diff
@@ -71,6 +77,22 @@ SiStripDigitizerAlgorithm::SiStripDigitizerAlgorithm(const edm::ParameterSet& co
   else LogDebug("SiStripDigitizerAlgorithm")<<" SingleStripNoise: OFF";
   if(CommonModeNoise) LogDebug("SiStripDigitizerAlgorithm")<<" CommonModeNoise: ON";
   else LogDebug("SiStripDigitizerAlgorithm")<<" CommonModeNoise: OFF";
+  
+  std::string line; 
+  APVProbaFile.open((APVProbabilityFile.fullPath()).c_str());
```

Comment:
@sdevissc - could this initialization be done only when needed according to the APVSaturationFromHIP value?